### ISSUE TITLE
WARNING: 00 _default: no tail silence word survived on the last frame, search failed

### ISSUE THREAD

**Issue by Jake-s16** [julius-speech/julius#111]
Just installed following the instructions. 
I get to point 4. test mozilla.wav.
I see lots of "WARNING: 00 _default: no tail silence word survived on the last frame, search failed" 
when I try to run ~/Software/julius-master/julius/./julius -C julius.jconf -dnnconf dnn.jconf in my 
~/Software/ENVR-v5.4.Dnn.Bin folder

**Comment by Estalhun** [comment]
I think the wave is too long. Try to cut into pieces shorter than 20 sec.

Jake-s16 <notifications@github.com> ezt írta (időpont: 2019. júl. 11., Cs
15:56):

**Comment by Jake-s16** [comment]
I agree but that really doesn't make sense. Why would someone package that wav file to be used if it was too long?

**Comment by kingbeowulf** [comment]
Same error here trying to test release 4.5 for distro packaging (BTW, needing <=20s speech segments would be very silly).

Following the Quick Start and README.md in ENVR-v5.4.Dnn.Bin.zip, I see a whole string of 
WARNING: 00 _default: no tail silence word survived on the last frame, search failed
<search failed>
errors.  
I compiled with "configure --enable-words-int --enable-sp-segment"  I've attached the output.


[test.txt](https://github.com/julius-speech/julius/files/3737529/test.txt)

**Comment by kingbeowulf** [comment]
> Same error here trying to test release 4.5 for distro packaging (BTW, needing <=20s speech segments would be very silly).
> 
> Following the Quick Start and README.md in ENVR-v5.4.Dnn.Bin.zip, I see a whole string of
> WARNING: 00 _default: no tail silence word survived on the last frame, search failed
> 
> errors.

Turns out ENVR-v5.4.Gmm.Bin.zip is a better choice to test and use release 4.5.

**Comment by programmin1** [comment]
I just got the same thing, when i set up in a new directory referencing the download on which I hadn't made the change to dnn.jconf:

```
feature_type MFCC_E_D_A_Z
feature_options -htkconf wav_config -cvn -cmnload ENVR-v5.3.norm -cvnstatic
num_threads 1
feature_len 48
context_len 11
input_nodes 528
output_nodes 7461
hidden_nodes 1536
hidden_layers 5
W1 ENVR-v5.3.layer2_weight.npy
W2 ENVR-v5.3.layer3_weight.npy
W3 ENVR-v5.3.layer4_weight.npy
W4 ENVR-v5.3.layer5_weight.npy
W5 ENVR-v5.3.layer6_weight.npy
B1 ENVR-v5.3.layer2_bias.npy
B2 ENVR-v5.3.layer3_bias.npy
B3 ENVR-v5.3.layer4_bias.npy
B4 ENVR-v5.3.layer5_bias.npy
B5 ENVR-v5.3.layer6_bias.npy
output_W ENVR-v5.3.layerout_weight.npy
output_B ENVR-v5.3.layerout_bias.npy
state_prior_factor 1.0
state_prior ENVR-v5.3.prior
state_prior_log10nize false
```
